app/docs/user.md

```diff
@@ -1,3 +1,195 @@
-# YourFunctionName
+# buckinghamPiTheorem
 
-Teacher-facing documentation for this function.
\ No newline at end of file
+Checks that the set of quantities in the response matches the set of quantities in the sense given by the Buckingham Pi theorem.
+
+There are three different ways of supplying this function with the necessary information.
+- In the answer, provide an example set of groups as a comma separated list. When used this way the function assumes that the given list is correct and contains at least the minimum number of groups.
+- In the `quantities` parameter, supply a list of what the dimensions for each quantity is and set answer to `-`. The function will then compute a list of sufficiently many independent dimensionless quantities and compare to the response.
+- In the `quantities` parameter, supply a list of what the dimensions for each quantity is and in the answer, supply a list of groups as in the first option. The function will then check that the supplied answer is dimensionless and has a sufficient number of independent groups before comparing it to the response.
+
+Note that in lists of groups the items should ideally be written in the form $q_1^{c_1} \cdot q_2^{c_2} \cdots q_n^{c_n}$ where $q_1, q_2 \ldots q_n$ are quantities and $c_1, c_2 \ldots c_n$ are integers, but the function can also handle item that are sums with terms written in the form $a \cdot q_1^{c_1} \cdot q_2^{c_2} \cdots q_n^{c_n}$ where $q_1, q_2 \ldots q_n$ are quantities, $c_1, c_2 \ldots c_n$ rational numbers and $a$ a constant. If the total number of groups is less than required the set of groups is considered invalid, even if there is a sufficient number of terms with independent power products in the response.
+
+## Inputs
+All input parameters need to be supplied via the **Grading parameters** panel.
```

Comment:
What are grading parameters? The word grading shouldn't appear. Maybe this is the parameters in the 'evaluation' panel of the 'configure' window for response areas?

Reply:
Presence of 'grading' is an artifact remaining from older version of documentation. Removed all found instances and generally removed specific references to the web client UI as it might change or the evaluation function might be used elsewhere.